tests/tests.targets

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
   <Target Name="FindTestDirectories">
     <ItemGroup>
-      <AllTestAssemblies Include="$(TestAssemblyDir)*.XUnitWrapper.dll" />
+      <AllTestAssemblies Include="$(BaseOutputPathWithConfig)\*\*.XUnitWrapper.dll" />
```

Comment:
You should probably keep TestAssemblyDir here and work by modifying it. The original intentions was to decouple them so that we can disable them from where the build outputs it to.

At the very least please ensure that SkipTestAssemblies works fine